### PR TITLE
Added my name (Joel)

### PR DESCRIPTION
**Pull request by Felanrod**
I added my name!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <header>
         <ul>
             <li>Jayden</li>
+            <li>Joel</li>
```
Comment (JaydenHluszko) [inline index.html:14]: You screwed it up

Comment (Felanrod) [reply index.html:14]: I changed it

Comment (Felanrod) [reply index.html:14]: Is that better?

Comment (JaydenHluszko) [reply index.html:14]: yep

Comment (Felanrod) [reply index.html:14]: I added after Andrew

**Pull request merged**